site/docs/guidelines/workflow.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Workflow
+navTitle: Workflow
+summaryParagraph: A workflow is a long task broken up into a series of smaller, linked steps.
+tags: ["Wizard", "Multi-step workflow", "Onboarding"]
+needToKnow:
+  - "Use good defaults everywhere possible."
+---
+
+import WhenToUseAndWhenNotToUse from "docs-components/WhenToUseAndWhenNotToUse"
+import WhenToUse from "docs-components/WhenToUse"
+import WhenNotToUse from "docs-components/WhenNotToUse"
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D49154%253A36853&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+## To keep in mind
+
+Breaking the long task into a series of smaller, linked steps, lets you:
+
+- Distribute the overall cognitive load
+- Help the user understand the process by breaking it into more understandable chunks
+- Guide the user through the process with minimal friction.
+- Allow decisions made in preceding steps to vary the workflow of subsequent steps 
+
+### Anatomy
+
+- Each **step** in a workflow contains a logical grouping of form elements needed to complete the overall task.
+- The **workflow header** includes the Culture Amp logo, step title, and exit link:
+    - The **step title** matches the step title in the workflow footer [progress stepper](/components/progress-stepper).
```

Comment:
What if the step title is too long for the footer? Is a shortened version of the title acceptable (e.g. ‘Import employee data’ in the title, ‘Import’ in the footer)